app/celery/tasks.py

```diff
@@ -450,18 +452,19 @@ def save_emails(self, service_id: Optional[str], signed_notifications: List[Any]
         notification_id = notification.get("id", create_uuid())
         notification["notification_id"] = notification_id
         reply_to_text = ""  # type: ignore
+
         if sender_id:
             reply_to_text = dao_get_reply_to_by_id(service_id, sender_id).email_address
-        if isinstance(template, tuple):
-            template = template[0]
+            if isinstance(template, tuple):
+                template = template[0]
         # if the template is obtained from cache a tuple will be returned where
         # the first element is the Template object and the second the template cache data
         # in the form of a dict
         elif isinstance(template, tuple):
             reply_to_text = template[1].get("reply_to_text")  # type: ignore
             template = template[0]
         else:
-            reply_to_text = template.get_reply_to_text()  # type: ignore
+            reply_to_text = template.get_reply_to_text()
```

Comment:
Maybe we don't need any of these `ignore`s? If the tests will pass without them that would be preferred.

Reply:
Might be best to supersede this PR now know the indentation is all that was required